### PR TITLE
minor improvements

### PR DESCRIPTION
**Pull request by Vahid-Galavi**
**Description**
Some minor improvements in GeoMechanics Application.

### CODE REVIEW

File: applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_quasistatic_U_Pw_scheme.hpp [hunk 1]
```diff
@@ -494,6 +494,7 @@ class NewmarkQuasistaticUPwScheme : public Scheme<TSparseSpace,TDenseSpace>
         mBeta = 0.25;
         mGamma = 0.5;
         mTheta = 0.5;
+        mDeltaTime = std::numeric_limits<double>::epsilon();
```
Comment (rubenzorrilla) [inline applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_quasistatic_U_Pw_scheme.hpp]: If this is to suppress the initialization warning I'd just set it to zero. If not, please ignore my comment.

**Review by rubenzorrilla — DISMISSED** [review]
Minor changes.

File: applications/GeoMechanicsApplication/tests/test_steady_state_groundwater_flow.py [hunk 2]
```diff
@@ -58,12 +58,12 @@ def calculate_outflow_discharge(self, simulation):
...
         hydraulic_discharge = test_helper.get_hydraulic_discharge(simulation)
-        sum = 0
+        sumValue = 0
         for value in hydraulic_discharge:
             if (value > 0):
-                sum = sum + value
+                sumValue = sumValue + value
 
-        return sum
+        return sumValue
```
Comment (philbucher) [inline applications/GeoMechanicsApplication/tests/test_steady_state_groundwater_flow.py]:
~~~py
from functools import reduce
return reduce(lambda a,b: a+b if b>0 else a, hydraulic_discharge, 0)
~~~

some more advanced python :)

File: applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_quasistatic_Pw_scheme.hpp [hunk 3]
```diff
@@ -66,7 +66,7 @@ class NewmarkQuasistaticPwScheme : public NewmarkQuasistaticUPwScheme<TSparseSpa
         BaseType::Check(rModelPart);
 
         //check that variables are correctly allocated
-        for (ModelPart::NodesContainerType::const_iterator it=rModelPart.NodesBegin(); it!=rModelPart.NodesEnd(); it++)
+        for (ModelPart::NodesContainerType::const_iterator it=rModelPart.NodesBegin(); it!=rModelPart.NodesEnd(); ++it)
```
Comment (philbucher) [inline applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_quasistatic_Pw_scheme.hpp]:
```suggestion
        for (const auto& r_node : rModelPart.Nodes())
```

even shorter

File: applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_quasistatic_U_Pw_scheme.hpp [hunk 4]
```diff
@@ -66,7 +66,7 @@ class NewmarkQuasistaticUPwScheme : public Scheme<TSparseSpace,TDenseSpace>
         BaseType::Check(rModelPart);
 
         //check that variables are correctly allocated
-        for (ModelPart::NodesContainerType::const_iterator it=rModelPart.NodesBegin(); it!=rModelPart.NodesEnd(); it++)
+        for (ModelPart::NodesContainerType::const_iterator it=rModelPart.NodesBegin(); it!=rModelPart.NodesEnd(); ++it)
```
Comment (philbucher) [inline applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_quasistatic_U_Pw_scheme.hpp]:
```suggestion
        for (const auto& r_node : rModelPart.Nodes())
```

even shorter

**Review by philbucher — DISMISSED** [review]

File: applications/GeoMechanicsApplication/tests/test_steady_state_groundwater_flow.py (hunk 2 above)
Comment (Vahid-Galavi) [reply applications/GeoMechanicsApplication/tests/test_steady_state_groundwater_flow.py]: @philbucher Thanks! I am using advanced python now ;)

File: applications/GeoMechanicsApplication/tests/test_steady_state_groundwater_flow.py [hunk 5]
```diff
@@ -58,12 +58,9 @@ def calculate_outflow_discharge(self, simulation):
...
 
         hydraulic_discharge = test_helper.get_hydraulic_discharge(simulation)
-        sumValue = 0
-        for value in hydraulic_discharge:
-            if (value > 0):
-                sumValue = sumValue + value
 
-        return sumValue
+        from functools import reduce
+        return reduce(lambda a,b: a+b if b>0 else a, hydraulic_discharge, 0)
```
Comment (philbucher) [inline applications/GeoMechanicsApplication/tests/test_steady_state_groundwater_flow.py:63]: @Vahid-Galavi I am quite sure it works as I tested it with a simpler example but please make sure you also check it once

**Review by philbucher — APPROVED** [review]

File: applications/GeoMechanicsApplication/tests/test_steady_state_groundwater_flow.py (hunk 5 above)
Comment (Vahid-Galavi) [reply applications/GeoMechanicsApplication/tests/test_steady_state_groundwater_flow.py:63]: @philbucher It works before pushing. It works. Thanks again for the suggestion.

**Pull request merged**